litellm/integrations/traceloop.py

```diff
@@ -1,10 +1,26 @@
+import traceback
+
+
 class TraceloopLogger:
     def __init__(self):
-        from traceloop.sdk.tracing.tracing import TracerWrapper
-        from traceloop.sdk import Traceloop
+        try:
+            from traceloop.sdk.tracing.tracing import TracerWrapper
+            from traceloop.sdk import Traceloop
+
+            Traceloop.init(app_name="Litellm-Server", disable_batch=True)
+            self.tracer_wrapper = TracerWrapper()
+        except Exception as e:
+            raise Exception(
+                f"\033[Traceloop not installed, try running 'pip install traceloop-sdk' to fix this error: {e}\n{traceback.format_exc()}\033[0m"
```

Comment:
You are missing `91m` characters before `Traceloop` in the error message causing the color to be white and missing `T`, check the image
<img width="844" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/5474da30-9110-4ee7-94ea-ce7cd7eba8e5">